apps/server/src/modules/tool/controller/dto/request/context-external-tool-context.params.ts

```diff
@@ -3,11 +3,11 @@ import { IsEnum, IsMongoId } from 'class-validator';
 import { ToolContextType } from '../../../interface';
 
 export class ContextExternalToolContextParams {
-	@ApiProperty({ nullable: false, required: true })
+	@ApiProperty({ nullable: false, required: true, example: '0000dcfbfb5c7a3f00bf21ab' })
```

Comment:
Nice to know that the example key exists.

Reply:
Yes, it makes it definitely easier to test :P but we should only use data from seeds